internal/providers/github/properties/release.go

```diff
@@ -0,0 +1,154 @@
+// SPDX-FileCopyrightText: Copyright 2024 The Minder Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package properties
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	go_github "github.com/google/go-github/v63/github"
+
+	"github.com/mindersec/minder/internal/entities/properties"
+	minderv1 "github.com/mindersec/minder/pkg/api/protobuf/go/minder/v1"
+	v1 "github.com/mindersec/minder/pkg/providers/v1"
+)
+
+// Release Properties
+const (
+	// ReleasePropertyOwner represents the github owner
+	ReleasePropertyOwner = "github/owner"
+	// ReleasePropertyRepo represents the github repo
+	ReleasePropertyRepo = "github/repo"
+	// ReleasePropertyTag represents the github release tag name.
+	ReleasePropertyTag = "github/tag"
+	// ReleasePropertyBranch represents the github release branch
+	ReleasePropertyBranch = "github/branch"
```

Comment:
It was suggested we should have a common-top-level property for these, and for the commit SHA.